identity-example/src/main/java/com/stripe/android/identity/example/MainActivity.kt

```diff
@@ -25,21 +27,44 @@ class MainActivity : AppCompatActivity() {
         Json {
             ignoreUnknownKeys = true
             isLenient = true
-            encodeDefaults = true
+            encodeDefaults = false
         }
     }
 
+    private lateinit var allowDL: CheckBox
+    private lateinit var allowPassport: CheckBox
+    private lateinit var allowID: CheckBox
+    private lateinit var requireIdNumber: CheckBox
+    private lateinit var requireMatchingSelfie: CheckBox
+    private lateinit var requireLiveCapture: CheckBox
```

Comment:
rather than defining these here could we instead just reference these by `binding.allowTypeDl` etc?

Reply:
Yes, updated